core/types/block.go

```diff
@@ -171,6 +179,9 @@ func EmptyHeader() *Header {
 		h.parentDeltaS[i] = big.NewInt(0)
 		h.number[i] = big.NewInt(0)
 	}
+	for i := 0; i < common.NumZonesInRegion; i++ {
+		h.primeEntropyThreshold[i] = big.NewInt(0)
```

Comment:
Instead of reallocating each time, can we reuse `common.big0`?

Reply:
just looked at it, there are so many places in that file that use big.NewInt(0), I can create an issue to clean all that up, there is no need for us to re declare the big Int 0 again and again

Reply:
ok, yeah, please create an issue. Allocating a `big.NewInt(0)` every time is costly. Potentially very costly if the allocator has to do a lot of work to find an allocatable memory range (which may be the case as our system scales to high TPS)